app/Http/Requests/BatchRequest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
```

Comment:
Please remove this importing `FormRequest` class.
We're extending our own FormRequest in this namespace.

Reply:
solved